opendebates/moderator_views.py

```diff
@@ -46,15 +47,12 @@ def mark_duplicate(request):
         votes_to_merge.update(original_merged_submission=to_remove, submission=duplicate_of)
 
     if request.POST.get("send_email") == "yes":
-        # FIXME: `emails` was not defined, so I have commented it out for now. --vkurup
         if request.POST.get("handling") == "merge":
-            pass
-            # emails.idea_is_merged(to_remove)
+            send_email("your_idea_is_merged", {"idea": to_remove})
```

Comment:
See tests -- I set up the recipient field (and all other info that's needed to send the mail) as a templated attribute on the `EmailTemplate` object.  So the `EmailTemplate` object plus the idea object can fully determine the email behavior.  The admins would use the Django Admin Site TTW to create an `EmailTemplate` with `recipient = {{ idea.voter.email }}`.